config/config.exs

```diff
@@ -34,3 +34,14 @@ config :ftp,
   password: "password1",
   ro_dirs: ["/home/alan/Development"],
   rw_dirs: ["/home/alan/Development/ftp"]
+
+config :ftp, :events,
```

Comment:
Why are these configured in the config file? Are these likely to change through environments? These seem like they should be part of the code and not the config. They are all over the bifrost code so I think that is where they should live.

Reply:
3 reasons.
1. For testing, it's quicker easier to add/remove events to here.
2. I think it is good to outline, in one place, all of the events that can be generated (could be done in a comment, I know).
3. Gives a user an easy option to subscribe to all the events in their own module. You will see what I mean when I push my PR in CC.

Reply:
Okay!